06_puppy/runnerdave/storer_test.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"testing"
+
+	tassert "github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	mem storerImpl = iota
+)
+
+var (
+	puppy1 = func() Puppy {
+		return Puppy{ID: 11, Breed: "Chihuahua", Colour: "Brown", Value: 12.30}
+	}
+)
+
+type storerImpl int
+
+type storerSuite struct {
+	suite.Suite
+	store Storer
+	impl  storerImpl
```

Comment:
I might have missed something, is `impl` field actually used here?

Reply:
good catch, cleaned it from the code